talent/offboarding.md

```diff
@@ -0,0 +1,74 @@
+This document describes the off-boarding process at source{d}.
+
+We believe that the off-boarding process should be smooth and clear in order to cause minimal disruption for each person.
+
+The relationship with the company can finish in two different ways: 
+
+#### The employee leaves voluntarily
+
+_This process describes employees working for the Spanish entity, for the USA this is not yet documented._
+
+- The employee has to inform the company, usually 15 days in advance. 
+- The employee has to sign a resignation letter with their information and the end date, as shown in the [example] (https://drive.google.com/file/d/0B6AGEF9RyVDxdEFULWx1dzNSd0E/view?usp=sharing).
```

Comment:
I could not find a resignation letter in that link.

Reply:
It's this one as far as I know: Papeleta de conciliación individual @dpordomingo